pootle/apps/pootle_comment/forms.py

```diff
@@ -52,13 +52,13 @@ def clean(self):
             raise CommentNotSaved(dict(comment=should_not_save))
 
     def save(self):
-        ob = self.comment
-        ob.user = self.cleaned_data["user"]
-        ob.submit_date = datetime.now()
-        ob.save()
+        obj = self.comment
```

Comment:
how about we make this one `comment` -  we dont really need a local var at all tbh tho